<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="mediaGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Media</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>551</width>
       <height>381</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="controlGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>1001</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QPushButton" name="pauseBtn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="playBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QPushButton" name="resumeBtn">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Resume</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopBtn">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="playlistGroup">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>411</width>
      <height>421</height>
     </rect>
    </property>
    <property name="title">
     <string>Playlist</string>
    </property>
    <widget class="QPushButton" name="removeBtn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
    <widget class="QListWidget" name="playList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>211</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addBtn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="upBtn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downBtn">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>330</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="txtEdit">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
